hera_qm/ant_metrics.py

```diff
@@ -210,12 +240,20 @@ def corr_cross_pol_metrics(corr_stats, xants=[]):
             # this loops over all the combinations of same and cross-pols
             for i, (pol1, pol2) in enumerate([(same_pols[0], cross_pols[0]), (same_pols[0], cross_pols[1]),
                                               (same_pols[1], cross_pols[0]), (same_pols[1], cross_pols[1])]):
-                if (a1, a2, pol1) in corr_stats and (a1, a2, pol2) in corr_stats:
-                    diffs[i].append(corr_stats[(a1, a2, pol1)]- corr_stats[(a1, a2, pol2)])
+                # reverse baselines if necessary
+                bl1 = (a1, a2, pol1)
```

Comment:
I think this method is double counting baselines, which isn't a problem, but also isn't necessary and makes the code a bit more complicated. I'm fine with leaving it this way if there's a reason you prefer it though.

Reply:
You're right, though in fairness I was trying to address the problem where your method quadruple counted: 
https://github.com/HERA-Team/hera_qm/blob/6d311421cf36dd49f40e6ca9277ee1468fd92229/hera_qm/ant_metrics.py#L337-L354
(keys has 4 baselines for every antenna number pair)

Let me try to see if I can figure out something more elegant.

Reply:
It should work if take what you have but just skip checking for reverse baselines - it will iterate over (a1,a2) and (a2,a1) so the iteration will take care of counting everything once.

Reply:
Ok, I thought about this a bit more. I don't think we can avoid the double counting unless we assume that if e.g. `(0, 1, 'ee')` is in `self.corr_stats` then `(0, 1, 'nn')`, `(0, 1, 'en')`, and `(0, 1, 'ne')` will also be in the array and not the baseline-reverse one. That's generally going to be true, but this handles the weird edge-case where that wouldn't be true.

I did find a way to make it one line shorter though.

Reply:
Ok, found a way to simplify the code using DataContainers. Just pushed.